app/src/main/java/org/oppia/app/onboarding/OnboardingListener.kt

```diff
@@ -0,0 +1,11 @@
+package org.oppia.app.onboarding
+
+/** Listener for buttons in OnboardingFragment */
+interface OnboardingListener {
```

Comment:
Hmm, 'OnboardingListener' sounds like it will indicate whenever onboarding state has changed, but that's not actually what's happening here. Can we name this more specifically? Maybe 'OnboardingNavigationListener'?

Reply:
Okay!